index.html

```diff
@@ -201,5 +202,58 @@ <h4>Skills</h4><br>
             </ul>
         </div> 
 </section> 
+<!--CONTACT PAGE STARTS HERE -->
+<!--FORM DESIGN STARTS HERE -->
+<section class="page">
+    <div class="gap"></div>
+    <div class="c-form">
+    <div class="f-header">
+        <h2 class="getStart">Get started</h2>
+        <p class="About">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it .</p>
+    </div>    
```

Comment:
- [ ] As I told I impressed by your design @Javed-Shahbaz , but on your mobile version there is something you have to change. Look at the design below and make sure that your section fill the screen as well. It makes the app more professional :+1: 

Your Design 

![Screenshot from 2022-12-08 17-17-57](https://user-images.githubusercontent.com/80458199/206488727-7adb7dd4-45d4-4f23-97e8-fc6f2ff47214.png)